ambry-store/src/main/java/com.github.ambry.store/StoreFindToken.java

```diff
@@ -52,23 +54,29 @@
   private final Offset offset;
   private final StoreKey storeKey;
   private final UUID sessionId;
+  private final UUID incarnationId;
   private long bytesRead;
 
   StoreFindToken() {
```

Comment:
Nope. Still we will have UnInitialized tokens being used at places. Atleast we can't get rid of the token initialization used in replication(StoreFindToken.getNewFindToken())